src/EllCrv/NewtonLift.jl

```diff
@@ -0,0 +1,302 @@
+#import Hecke.AbstractAlgebra
+export newton_lift
+add_verbosity_scope(:NewtonIteration)
+
+_valuation(x::RingElem) = iszero(x) ? QQ(precision(parent(x))) : valuation(x)
+
+
+r"""
```

Comment:
```suggestion
raw"""
```
Also here. `r""` produces a regex apparently.